myui/__init__.py

```diff
@@ -6,7 +6,11 @@
 import tornado.ioloop
 import tornado.options
 from types import ModuleType
+import logging
 
+access_log = logging.getLogger("tornado.access")
+app_log = logging.getLogger("tornado.application")
+# gen_log = logging.getLogger("tornado.general")
```

Comment:
Decided not to use it, it's more of an internal logger used by tornado, but we may want it in the future so I left it in as more of a note